### ISSUE TITLE
FEATURE: Adapt Dockerfile to use Entrypoint for execution-dependent tasks

### ISSUE THREAD

**Issue by rshad** [wazuh/wazuh-qa#233]
Hi all!

**Problem Description**

When building docker image, using the [dockerfile](https://github.com/wazuh/wazuh-qa/blob/devel/kitchen/wazuh-chef/manager/files/dockerfile) of `wazuh-chef` pipeline, it's taking a lot of time on each image build as we need to install the required Ruby gems, then they are installed on each build.

This non-efficient way of working would be avoided by following a different approach, we describe as follows:

- [ ] Download the target image `ruby` only once.
- [ ] Install the Gems only once.
- [ ] Run as ENTRYPOINT the execution-dependent tasks.

**`NOTE`**: This description may be updated in the future if needed.

Kr,

Rshad

**Comment by rshad** [comment]
Duplicate of https://github.com/wazuh/wazuh-qa/issues/235
